packages/firestore/src/util/log.ts

```diff
@@ -28,8 +28,8 @@ export function getLogLevel(): LogLevel {
   return logClient.logLevel;
 }
 
-export function setLogLevel(newLevel: LogLevel): void {
-  logClient.logLevel = newLevel;
+export function setLogLevel(newLevel: LogLevelString | LogLevel): void {
```

Comment:
You can probably drop `LogLevel` as an input type.

Reply:
async_queue.test.ts calls `getLogLevel()` before changing it, then set the saved log level back when test is done. https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/test/unit/util/async_queue.test.ts#L84

It's only testing code, but i think it does no harm to keep it anyways?

Reply:
SGTM.

This could also be fixed by returning the LogLevelString from `getLogLevel()` and calling `setLogLevel('silent')` instead of `setLogLevel(LogLevel.SILENT)`. We can leave as is though.